ui/apps/platform/src/services/ComplianceResultsService.ts

```diff
@@ -0,0 +1,223 @@
+import axios from 'services/instance';
+import qs from 'qs';
+
+import { SearchFilter, ApiSortOption } from 'types/search';
+import { getListQueryParams, getRequestQueryStringForSearchFilter } from 'utils/searchUtils';
+
+import { complianceV2Url } from './ComplianceCommon';
+
+const complianceResultsBaseUrl = `${complianceV2Url}/scan`;
+
+export const ComplianceCheckStatusEnum = {
+    UNSET_CHECK_STATUS: 'UNSET_CHECK_STATUS',
+    PASS: 'PASS',
+    FAIL: 'FAIL',
+    ERROR: 'ERROR',
+    INFO: 'INFO',
+    MANUAL: 'MANUAL',
+    NOT_APPLICABLE: 'NOT_APPLICABLE',
+    INCONSISTENT: 'INCONSISTENT',
+} as const;
+
+export type ComplianceCheckStatus =
+    (typeof ComplianceCheckStatusEnum)[keyof typeof ComplianceCheckStatusEnum];
+
+type ComplianceScanCluster = {
+    clusterId: string;
+    clusterName: string;
+};
+
+export type ComplianceCheckStatusCount = {
+    count: number;
+    status: ComplianceCheckStatus;
+};
+
+export type ComplianceScanStatsShim = {
+    scanName: string;
+    checkStats: ComplianceCheckStatusCount[];
+    lastScan: string; // ISO 8601 date string
+};
```

Comment:
Do we need `scanConfigId: string;` property here?
https://github.com/stackrox/stackrox/blob/master/proto/api/v2/compliance_results_service.proto#L72

Reply:
Same question for `ComplianceScanResult` type below.

Reply:
Updated